routes/api-routes.js

```diff
@@ -12,6 +12,18 @@ module.exports = app => {
       });
   });
 
+  app.get("/api/getProfile/:profileId", (req, res) => {
+    db.User.findOne({ where: { profileId: req.params.profileId } }).then(
+      userResults => {
+        db.Customer.findOne({ where: { userId: userResults.id } }).then(
+          profileResults => {
+            res.json(profileResults);
+          }
+        );
+      }
+    );
+  });
```

Comment:
Please make changes discussed

Reply:
Changes done. removed the query to user model